src/frontends/pytorch/src/op/inverse.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "openvino/op/inverse.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_inverse(const NodeContext& context) {
+    // aten::inverse(Tensor self) -> Tensor
+    // aten::inverse(Tensor self, *, Tensor out) -> Tensor
+    num_inputs_check(context, 1, 2);
+    const auto input_tensor = context.get_input(0);
+    const auto input_type = input_tensor.get_element_type();
+
+    ov::Output<ov::Node> result;
+    if (input_type == element::bf16 || input_type == element::f16 || input_type == element::f32) {
```

Comment:
shoud checking input_dtype.is_real() enough here?

Reply:
As of now, CPU does not support f64 explicitly, so it is also converted to f32 in order to preserve low latency on inference, hence this manual type check.